crates/ra_ide/src/diagnostics.rs

```diff
@@ -362,6 +383,54 @@ mod tests {
         assert_eq!(diagnostics.len(), 0, "unexpected diagnostics:\n{:#?}", diagnostics);
     }
 
+    /// Takes a multi-file input fixture with annotated cursor position and the list of disabled diagnostics,
+    /// and checks that provided diagnostics aren't spawned during analysis.
+    fn check_disabled_diagnostics(
+        ra_fixture: &str,
+        disabled_diagnostics: impl IntoIterator<Item = String>,
+    ) {
+        let disabled_diagnostics: std::collections::HashSet<_> =
```

Comment:
Suggest importing `HashSet` instead

```suggestion
        let disabled_diagnostics: HashSet<_> =
```